justfile

```diff
@@ -63,6 +63,22 @@ check_v2 *FLAGS:
     cargo check {{ check_flags }} --features "${FEATURES}"  {{ FLAGS }}
     set +x
 
+run_v2 *FLAGS:
+    #! /usr/bin/env bash
+    set -euo pipefail
+
+    FEATURES="$(cargo metadata --all-features --format-version 1 | \
+        jq -r '
+            [ ( .workspace_members | sort ) as $package_ids # Store workspace crate package IDs in `package_ids` array
+            | .packages[] | select( IN(.id; $package_ids[]) ) | .features | keys[] ] | unique # Select all unique features from all workspace crates
+            | del( .[] | select( any( . ; . == ("v1") ) ) ) # Exclude some features from features list
```

Comment:
We can't do this, since it would still include features like AWS KMS, key manager service, etc.